ci/helm-chart/Chart.yaml

```diff
@@ -15,9 +15,9 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 2.6.0
+version: 3.0.0
```

Comment:
[This PR](https://github.com/coder/code-server/pull/5285) was identified as a breaking change, hence the major bump here.